src/common/constants.ts

```diff
@@ -9,6 +9,9 @@ export const deriv_com_app_id = 16929
 export const deriv_me_app_id = 1411
 export const deriv_be_app_id = 30767
 
+export const supported_domains = [deriv_com_url, deriv_me_url]
+export const domains = [...supported_domains, 'localhost', 'deriv-com-git-fork'] //deriv-com-git-fork for vercel server, localhost - for developer mode
+
 const domain_url_pair = {
     [deriv_com_url]: deriv_com_url,
     [deriv_me_url]: deriv_me_url,
```

Comment:
Could we consolidate these? The `domain_url_pair` was actually to replace `supported_domains`.